core/src/components/avatar/test/basic/avatar.e2e.ts

```diff
@@ -0,0 +1,22 @@
+import { expect } from '@playwright/test';
+import { test } from '@utils/test/playwright';
+
+test.describe('avatar: basic', () => {
+  test('should not have visual regressions', async ({ page, skip }) => {
+    skip.rtl();
```

Comment:
I feel like we should include RTL tests here since it affects the chip and item screenshots. Maybe the screenshots of just the `ion-avatar` could be split into a different test, and RTL skipped there only?

Reply:
Hmm, makes sense to me! Could you add a comment to that effect here?